docs/fleet/fragments.md

```diff
@@ -28,95 +32,46 @@ Once you are confident that your configuration change works as expected, you can
-- Select **Insert fragment**.
-  Now, you can see the available fragments to add:
-
-  {{<imgproc src="appendix/try-viam/rover-resources/fragments/fragments_list.png" resize="400x" style="max-width: 500px" alt="List of available fragments">}}
-
-- Look through the list of available fragments.
-  Click on a fragment to open its menu.
-  Click **View JSON** to view the JSON configuration the fragment contains, and click **Insert fragment** to insert the fragment into your machine's configuration.
-- Click **Save** in the upper right corner of the page to save your new configuration.
-
-The components and services included in the fragment will now appear as _read-only_ cards on the **CONFIGURE** tab, along with a card for your fragment:
-
-{{<imgproc src="appendix/try-viam/rover-resources/fragments/fragments_cards.png" resize="400x" style="max-width: 500px" alt="List of available fragments">}}
-
-In the `JSON` configuration, you will see the fragment ID in the `fragments` section:
-
-```json
-{
-  "components": [],
-  "fragments": ["3e8e0e1c-f515-4eac-8307-b6c9de7cfb84"]
-}
-```
-
-For an example of adding a fragment to a machine, see the [Add a Rover Fragment to your Machine](/appendix/try-viam/rover-resources/rover-tutorial-fragments/).
-
 ## Modify the config of a machine that uses a fragment
 
-The configuration cards of components and services included in the fragment are _read-only_, meaning they cannot be modified from the fragment specification.
-
-Likewise, when you modify the fragment itself, those changes are pushed to all machines that use that fragment.
+The configuration of components and services included in a fragment are _read-only_.
+Likewise, when you modify the fragment itself, any changes are pushed to all machines that use that fragment.
 
 If you need to modify the config of just one machine that uses a fragment, you have two options:
 
 - Use `fragment_mods` in your machine's config to overwrite certain fields of the fragment.
 - Copy and paste the contents of the fragment, remove the link to the fragment itself, then modify the config as needed.
   - If you use this method, future updates to the fragment _will not_ be automatically pushed to your machine.
 
-### Use `fragment_mods`
-
-You can modify fragment fields in your machine's raw JSON config by using [update operators](https://www.mongodb.com/docs/manual/reference/operator/update/positional/#---update-).
-Viam supports all update operators except for `$setOnInsert`, `$`, `$[]`, and `$[<identifier>]`.
-
 {{% alert title="Support Notice" color="note" %}}
 
 `fragment_mods` are _not_ supported for the modification of [trigger](/configure/triggers/) configuration.
 You can create a trigger with a fragment but you cannot modify it with `fragment_mods`.
 
 {{% /alert %}}
 
-To configure fragment mods:
+### Use `fragment_mods`
```

Comment:
```suggestion
### Overwrite fragments to add machine-specific modifications
```
I think overwrites is what the feature is called rather than fragment_mods. @nataliajacobowitz - thought?

Reply:
In fact you could also remove this and the revert fragment mods titles.